src/main/kotlin/org/rust/ide/intentions/import/ImportNameIntention.kt

```diff
@@ -151,6 +152,56 @@ class ImportNameIntention : RsElementBaseIntentionAction<ImportNameIntention.Con
         return !(element.parent is RsMembers && element.ancestorStrict<RsTraitItem>() != null)
     }
 
+    private fun filterUselessImports(candidates: List<ImportCandidate>, fileAttributes: RsFile.Attributes): List<ImportCandidate> {
+        val finalCandidates = mutableListOf<ImportCandidate>()
+        val stdlibCandidates = mutableListOf<ImportCandidate>()
+        val otherCandidates = mutableListOf<ImportCandidate>()
```

Comment:
Why we need `otherCandidates` here? We can just add "other candidates" directly to `finalCandidates`